### PR TITLE
Fix demisto.log in content

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Fix demisto.log in content code so the new linter validation will not fail.

### CODE REVIEW

File: Packs/AWS-ACM/ReleaseNotes/1_1_17.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### AWS - ACM
+- Replaced `demisto.log` with `demisto.debug`.
```
Comment (dantavori) [inline Packs/AWS-ACM/ReleaseNotes/1_1_17.md:4]:
```suggestion
- Replaced unwanted usage of *demisto.log()* with *demisto.debug()*.
```
let's change everywhere like this

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 2]
```diff
@@ -7857,7 +7857,7 @@ def is_demisto_version_ge(version, build_number=''):
         raise
     except ValueError:
         # dev editions are not comparable
-        demisto.log(
+        demisto.log(  # pylint: disable=E9012
```
Comment (dantavori) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:7860]: let's not ignore this one, and change it to demisto.error

File: Packs/Base/ReleaseNotes/1_31_6.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### CommonServerPython
+- Ignored `demisto.log` pylint error.
```
Comment (dantavori) [inline Packs/Base/ReleaseNotes/1_31_6.md:4]:
```suggestion
- Resolved *demisto.log()* XSOAR linter errors.
```

File: Packs/CommonScripts/ReleaseNotes/1_7_10.md [hunk 4]
```diff
@@ -2,4 +2,4 @@
 #### Scripts
 ##### GenerateRandomString
 - Improved implementation with limiting the maximum length of the generated string to be 10,000.
-- Updated the Docker image to `3.10.4.30607`.
\ No newline at end of file
+- Updated the Docker image to `3.10.4.30607`.
```
Comment (dantavori) [inline Packs/CommonScripts/ReleaseNotes/1_7_10.md:5]: please revert

File: Packs/Cymptom/Integrations/Cymptom/Cymptom.py [hunk 5]
```diff
@@ -240,7 +240,7 @@ def main():
 
     # Log exceptions
     except Exception as e:
-        demisto.log(str(e))
+        demisto.log(str(e))  # pylint: disable=E9012
```
Comment (dantavori) [inline Packs/Cymptom/Integrations/Cymptom/Cymptom.py:243]: why we are not replacing to debug here?

File: Packs/Digital_Defense_FrontlineVM/Integrations/Digital_Defense_FrontlineVM/Digital_Defense_FrontlineVM.py [hunk 6]
```diff
@@ -428,7 +428,7 @@ def get_host_id_from_ip_address(ip_address):
     if len(hosts_with_given_ip) < 1:
         msg = 'Host not found within Frontline.Cloud given host IP Address. Host will not be included in querying vulnerabilities'
         demisto.error('Frontline.Cloud get_host_id_from_ip_address -- ' + msg)  # print to demisto log in ERROR
-        demisto.log('Frontline.Cloud get_host_id_from_ip_address -- ' + msg)    # print to war room
+        demisto.log('Frontline.Cloud get_host_id_from_ip_address -- ' + msg)  # pylint: disable=E9012
```
Comment (dantavori) [inline Packs/Digital_Defense_FrontlineVM/Integrations/Digital_Defense_FrontlineVM/Digital_Defense_FrontlineVM.py:431]: same q here and for the followings

**Review by dantavori — APPROVED** [review]

File: Packs/Base/ReleaseNotes/1_31_6.md (hunk 3 above)
Comment (yaakovpraisler) [reply Packs/Base/ReleaseNotes/1_31_6.md]: done

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py (hunk 2 above)
Comment (yaakovpraisler) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:7860]: done

Comment (yaakovpraisler) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:7860]: Revertig to demisto.log and ignoring as it fails unit tests.

**Comment by yaakovpraisler** [comment]
Validation fails on missing test-playbooks/readme.
@ShahafBenYakir - Need a force merge.

**Pull request merged**